Emby.Server.Implementations/LiveTv/TunerHosts/M3uParser.cs

```diff
@@ -255,19 +251,14 @@ private string GetChannelNumber(string extInf, Dictionary<string, string> attrib
 
         private static bool IsValidChannelNumber(string numberString)
         {
-            if (string.IsNullOrWhiteSpace(numberString) ||
-                string.Equals(numberString, "-1", StringComparison.OrdinalIgnoreCase) ||
-                string.Equals(numberString, "0", StringComparison.OrdinalIgnoreCase))
-            {
-                return false;
-            }
-
-            if (!double.TryParse(numberString, NumberStyles.Any, CultureInfo.InvariantCulture, out _))
+            if (string.IsNullOrWhiteSpace(numberString)
+                || string.Equals(numberString, "-1", StringComparison.OrdinalIgnoreCase)
```

Comment:
I know this is just a formatting change, but IgnoreCase for -1 and 0 doesn't seem necessary